source/appendices/e-audit-requirements-and-tests/04-dr-tests.rst

```diff
@@ -0,0 +1,66 @@
+.. The COUNTER Code of Practice Release 5 © 2017-2021 by COUNTER
+   is licensed under CC BY-SA 4.0. To view a copy of this license,
+   visit https://creativecommons.org/licenses/by-sa/4.0/
+
+E.4 Database Report Audit Tests
+-------------------------------
+
+This section of the appendix outlines tests that MUST be run during audits for a Host_Type that is required to offer the Database Report.
+
+The Database Report will be COUNTER-compliant if the following audit tests are passed. The Database Search and Item Usage (DR_D1) and Database Access Denied (DR_D2) Standard Views of the Database Report will be COUNTER-compliant if the metrics match those in the Database Report, with the appropriate aggregation.
+
+Audit test requirements vary depending on the set up of the platform and database(s), as indicated by the Options within the tests below.
+
+Database Report tests apply to Host_Types A&I_Database, Aggregated_Full_Content, Discovery_Service, eBook_Collection, Full_Content_Database, and Multimedia_Collection.
+
+E.4.1 Searches_Regular and Searches_Automated
+"""""""""""""""""""""""""""""""""""""""""""""
+
+**Option 1**: Platform has multiple databases and the user can search the whole platform, multiple selected databases, or a single database.
+
+The auditor MUST run 100 searches on the platform
+
+* 50 searches against only 1 selected database, resulting in 50 Searches_Regular against that database.
+* 25 searches against 2 selected databases, resulting in 25 Searches_Regular against both databases.
+* 25 searches against all databases on the platform, resulting in 25 Searches_Regular against every database.
+
+**Option 2**: Platform has multiple databases and the user can search the whole platform or a single database.
+
+The auditor MUST run 100 searches on the platform
+
+* 50 searches against only 1 selected database, resulting in 50 Searches_Regular against that database.
+* 50 searches against all databases on the platform, resulting in 50 Searches_Regular against every database.
+
+**Option 3**: Platform has multiple databases but the user can only search the whole platform.
+
+The auditor MUST run 100 searches, resulting in 100 Searches_Automated (1 per search).
+
+**Option 4**: Platform has multiple databases but the user can only search a single database at a time.
+
+The auditor MUST run 100 searches across at least 10% of the available databases, restricted to one database at a time, resulting in 100 Searches_Regular (1 per search).
+
+**Option 5**: Platform has only one database.
+
+The auditor MUST run 50 searches, resulting in 50 Searches_Regular (1 per search).
+
+
+E.4.2 Total_Item_Requests and Unique_Item_Requests
```

Comment:
The numbers in this and the next section are only correct if whole book downloads with identifiable segments can be avoided.